src/utils/request.ts

```diff
@@ -11,6 +11,7 @@ export const post = async (
   options: optionsType = {}
 ) => {
   try {
+    leaveBreadcrumb(`POST ${url}`, { body }, "request");
```

Comment:
In `authenticateIfSuccessfulLink` we call this function after we get a response. Why not do the same for `post`?

Reply:
<img width="1031" alt="image" src="https://user-images.githubusercontent.com/4605522/172352270-b0e907b6-289d-4cf7-9cf4-b637a667411e.png">

It happens automatically. 

Come to think of it this extra logging is probably a bad idea for post. We rarely use post but in the situations that we do we either send auth data or call some rest endpoint. Logging auth data including emails and passwords is a bad idea. (Although in this case it never gets hit in prod since we use okta).  I'll remove the POST logging altogether.